topology/probes/netlink.go

```diff
@@ -79,6 +79,26 @@ type NetLinkProbe struct {
 	wg      sync.WaitGroup
 }
 
+// RouteTable describes a list of Routes
+type RoutingTable struct {
+	Id      int
+	IfIndex int
+	Src     net.IP
+	Routes  []Route
+}
+
+// Route describes a route
+type Route struct {
+	Prefix   net.IP
+	Nexthops []NextHop
```

Comment:
I'm not able to see the nexthops in the WebUI for instance

Reply:
it was displayed only if it has multipath, now it fixed. thanks for pointing it out.